FirebaseMessaging/Sources/Token/FIRMessagingTokenManager.m

```diff
@@ -655,42 +653,51 @@ - (void)setAPNSToken:(NSData *)APNSToken withUserInfo:(NSDictionary *)userInfo {
 
   // Re-fetch any invalidated tokens automatically, this time with the current APNs token, so that
   // they are up-to-date.
-  if (invalidatedTokens.count > 0) {
+  // Or this is a fresh install and no apns token stored yet.
```

Comment:
Nit: can you include this comment as part of the comment on the previous line?

Reply:
Done.